### ISSUE TITLE
Priorities: intermediaries without priority support

### ISSUE THREAD

**Issue by kixelated** [httpwg/http-extensions#1734]
The draft uses the `Priority` header, which is end-to-end, to describe prioritization behavior that primarily applies hop-to-hop. There's a problem with this approach when an intermediary does not support prioritization.

For example, my origin supports prioritization but the CDN edge does not (and does not strip unknown headers).  My origin will receive requests with the `Priority` header, will create responses with it's own `Priority` header, and will write the responses based on this priority scheme.

The issue is that the intermediate will blindly forward the `Priority` header despite not actually prioritizing the response. If the path from origin to CDN edge is congested, then everything is actually okay. However, if the congestion exists between the CDN edge and the browser (most common), then response `Priority` header is lying.

This is especially important when determining priority support. My application changes it's behavior once it determines the server supports prioritization, as it can now safely issue multiple requests in parallel. However, this end-to-end parameter is actually not correct.

**Comment by kixelated** [comment]
And yeah, if there's any form of capabilities exchange or prioritization confirmation, it needs to be hop-by-hop. It's far too easy to accidentally use an intermediary that effectively disables prioritization. For example, spinning up a service in AWS and using an ALB (layer 7) instead of an NLB (layer 3).

**Comment by LPardue** [comment]
> Okay, it makes more sense that the response header is a hint for intermediaries. The draft should state that the client SHOULD NOT use the header as an acknowledgement that any prioritization occurred.
> 
> Relevant section:
> 
> > The Priority HTTP header field can appear in requests and responses. A client uses it to specify the priority of the response. A server uses it to inform the client that the priority was overwritten. An intermediary can use the Priority information from client requests and server responses to correct or amend the precedence to suit it (see {{merging}}).
> 
> The current text suggests the opposite; the server is correcting the client and telling it that prioritization did occur.

Agreed, this text had some issues. PTAL at https://github.com/httpwg/http-extensions/pull/1731/files#diff-4dc65e5537b9f2d8c77d359e6b7a032db79f294b488d2ae2b0554aa96f9b9258R377 and let me know if you think the change address your comment.
